src/ProgressOnderwijsUtils/Data/SiteBase.cs

```diff
@@ -11,13 +11,13 @@ public virtual object GetService(Type serviceType)
         public virtual IComponent Component
             => throw new NotSupportedException();
 
-        public virtual IContainer? Container
-            => null;
+        public virtual IContainer Container
+            => null!;

```

Comment:
hmm, moeten we hier een nep container voor maken oid?  Zou `throw new NotImplementedException()` werken?

Reply:
Jij hebt niet expliciet null gekozen hier?

Reply:
Ik heb hier vast null gekozen omdat het makkelijk was, en in praktijk wel nullable.  Meer zal daar niet achter zitten.

Reply:
(Dus geen reden wmb om null te laten)

Reply:
Blijkbaar werkt het niet als je een exception throwt.

Reply:
ja, ik zag het :-/  Naja, en lege container maken met wat lege methods is niet gestoord.

Reply:
maar wellicht overbodig, want we gaan zelf nooit de container uitlezen.

Reply:
Jij hebt voorkeur voor `null!` boven nep-container?

Reply:
Ik heb een afkeer van code die niets doet.

Reply:
naja, ik heb een afkeer voor foutieve null annotaties die ooit gekke dingen zouden kunnen doen. en een "leeg" null object, tja,... het zal.

Maar goed, ik hou je niet tegen, ik snap je punt wel.